internal/table/table.go

```diff
@@ -133,51 +140,120 @@ func rescaleColumns(colWidths []int, targetTotal int) {
 	colWidths[len(colWidths)-1] += targetTotal - mathutils.Total(colWidths...)
 }
 
+func getCroppedText(text string, maxLen int) []entry {
+	entries := make([]entry, 0)
+	pos := 0
+	matches := colorize.ColorRx.FindAllSubmatchIndex([]byte(text), -1)
+	runeText := []rune(text)
+
+	for pos < len(runeText) {
+		buffer := bytes.Buffer{}
```

Comment:
Why make a bytes buffer if you're just going to send it nothing but strings, and then convert it to a string at the end?

Reply:
Yes, not necessary